code/modules/surgery/organs/external/tails.dm

```diff
@@ -104,6 +112,17 @@
 	bodypart_overlay = /datum/bodypart_overlay/mutant/tail/cat
 
 	wag_flags = WAG_ABLE
+	sprite_to_use = "Cat"
+
+/obj/item/organ/external/tail/cat/bunny
```

Comment:
Now that they're not available to felinids is it important for these to be a subtype of cat tails?
Willing to believe the answer is yes, just double checking.